architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator (MCE). The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available in two ways:
+
+* As a standalone operator that you install as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription
+* As part of link:https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes[Red Hat Advanced Cluster Management for Kubernetes]
+
+[id="mce-on-ocp"]
+== Cluster management with multicluster engine on {product-title}
+
+When you enable multicluster engine on {product-title}, you gain the following capabilities:
+
+* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which is based on the HyperShift project. With hosted control planes, you have a centralized control plane for OpenShift clusters so that you can operate clusters in a hyperscale manner.
+* Hive, which provisions self-managed OpenShift clusters to the hub and completes the initial configurations for those clusters.
```

Comment:
```suggestion
* Hive, which provisions self-managed {product-title} clusters to the hub and completes the initial configurations for those clusters.
```

I'm not familiar with this area of the documentation - is "the hub" a well known term here or should it be described more specifically here?

Reply:
I believe "the hub" will be understood by customers who are familiar with cluster management.